langfuse-core/src/index.ts

```diff
@@ -87,8 +87,75 @@ class LangfuseFetchNetworkError extends Error {
   }
 }
 
+function isLangfuseFetchHttpError(error: any): error is LangfuseFetchHttpError {
+  return typeof error === "object" && error.name === "LangfuseFetchHttpError";
+}
+
+function isLangfuseFetchNetworkError(error: any): error is LangfuseFetchNetworkError {
+  return typeof error === "object" && error.name === "LangfuseFetchNetworkError";
+}
+
 function isLangfuseFetchError(err: any): boolean {
-  return typeof err === "object" && (err.name === "LangfuseFetchHttpError" || err.name === "LangfuseFetchNetworkError");
+  return isLangfuseFetchHttpError(err) || isLangfuseFetchNetworkError(err);
+}
+
+// Constants for URLs
+const SUPPORT_URL = "https://langfuse.com/support";
+const API_DOCS_URL = "https://api.reference.langfuse.com";
+const RBAC_DOCS_URL = "https://langfuse.com/docs/rbac";
+const RATE_LIMITS_URL = "https://langfuse.com/faq/all/api-limits";
+const NPM_PACKAGE_URL = "https://www.npmjs.com/package/langfuse";
+
+// Error messages
+const updatePromptResponse = `Make sure to keep your SDK updated, refer to ${NPM_PACKAGE_URL} for details.`;
+const defaultErrorResponse = `Unexpected error occurred. Please check your request and contact support: ${SUPPORT_URL}.`;
+
+// Error response map
+const errorResponseByCode = new Map<number, string>([
+  // Internal error category: 5xx errors, 404 error
+  [500, `Internal server error occurred. Please contact support: ${SUPPORT_URL}`],
+  [501, `Not implemented. Please check your request and contact support: ${SUPPORT_URL}.`],
+  [502, `Bad gateway. Please try again later and contact support: ${SUPPORT_URL}.`],
+  [503, `Service unavailable. Please try again later and contact support if the error persists: ${SUPPORT_URL}.`],
+  [504, "Gateway timeout. Please try again later and contact support: ${SUPPORT_URL}."],
+  [
+    404,
+    `Internal error occurred. Likely caused by race condition, please escalate to support if seen in high volume: ${SUPPORT_URL}`,
+  ],
+
+  // Client error category: 4xx errors, excluding 404
+  [
+    400,
+    `Bad request. Please check your request for any missing or incorrect parameters. Refer to our API docs: ${API_DOCS_URL} for details.`,
+  ],
+  [401, `Unauthorized. Please check your public/private host settings.`],
```

Comment:
Can we add a link here pointing to how to set up the SDK with the correct configs?